.gitignore

```diff
@@ -37,3 +37,66 @@ Carthage
 
 *.sublime-project
 *.sublime-workspace
+
+# NPM
+node_modules
+
+# @react-native/codegen
+/packages/react-native/React/FBReactNativeSpec/
+/packages/react-native-codegen/lib
+/packages/react-native-codegen/tmp/
+/packages/react-native/ReactCommon/react/renderer/components/rncore/
+/packages/rn-tester/NativeModuleExample/ScreenshotManagerSpec*
+/**/RCTThirdPartyFabricComponentsProvider.*
+
+# @react-native/codegen-typescript-test
+/packages/react-native-codegen-typescript-test/lib
+
+# Additional SDKs
+/packages/react-native/sdks/download
+/packages/react-native/sdks/hermes
+/packages/react-native/sdks/hermesc
+/packages/react-native/sdks/hermes-engine/hermes-engine-from-local-source-dir.tar.gz
+
+# Visual Studio Code (config dir - if present, this merges user defined
+# workspace settings on top of react-native.code-workspace)
+/.vscode
+
+# Visual Studio
+.vs
+
+# Android memory profiler files
+*.hprof
+
+# Temporary files created by Metro to check the health of the file watcher
+.metro-health-check*
+
+# CircleCI
+.circleci/generated_config.yml
+
+# Jest Integration
+/jest/integration/build/
+
+# Test Reports
+/reports
+
+# Stack Dumps generated when programs crash (Ex. bash.exe.stackdump on Win)
+*.stackdump
+
+# ReactCommon subdir shouldn't have Xcode project
+/packages/react-native/ReactCommon/**/*.xcodeproj
```

Comment:
Can we just gitignore the entire `/packages` directory? I assume these packages and their exact directories may change in the future. Also is there any need to include other things from `/packages` ever?